tox.ini

```diff
@@ -22,6 +22,14 @@ deps =
     zope.proxy
     zope.security
 
+[testenv:pypy3]
+# On certain platforms, at least OS X, pypy3 does not correctly read
+# a LANG environment variable; when cpython3.2 would return 'utf-8'
+# from locale.getpreferredencoding(), pypy3 returns 'us-ascii', causing
+# two test failures. Fix that by setting something it does read.
```

Comment:
Are you sure that wasn't merely a difference in tox versions?  tox 2.0.0 and 2.0.1 remove LANG (and LC_CTYPE, and LC_ALL, and in fact *EVERYTHING*) from the environment, which makes the test see an US-ASCII locale always.

tox 2.0.2 gave in to the pressure and added `LANG` to the default `passenv`.